src/proxy-ui-api/frontend/src/views/AddClient/SelectClientDialog.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <v-dialog :value="dialog" width="750" scrollable persistent>
+    <v-card class="xrd-card">
+      <v-card-title>
+        <span class="headline">{{$t("wizard.client.addClient")}}</span>
+        <v-spacer />
+        <i @click="cancel()" data-test="x-close-button"></i>
+      </v-card-title>
+
+      <v-card-text style="height: 500px;" class="elevation-0">
+        <v-text-field
+          v-model="search"
+          :label="$t('wizard.client.member')"
+          single-line
+          hide-details
+          class="search-input"
+          data-test="client-search-input"
+        >
+          <v-icon slot="append">mdi-magnify</v-icon>
+        </v-text-field>
+
+        <!-- Table -->
+        <v-radio-group v-model="selectedMember">
+          <table class="xrd-table members-table fixed_header">
+            <thead>
+              <tr>
+                <th class="checkbox-column"></th>
+                <th>{{$t('name')}}</th>
+                <th>{{$t('localGroup.id')}}</th>
+              </tr>
+            </thead>
+            <tbody v-if="selectableClients && selectableClients.length > 0">
```

Comment:
I think you can use optional chaining here

Reply:
Optional chaining here breaks the build. Not sure why, but probably because it's in the template.

Reply:
Ok, lets leave it then. I guess it is because of Vues template engine